moto/route53/responses.py

```diff
@@ -382,3 +391,12 @@ def list_or_change_tags_for_resource_request(self, request, full_url, headers):
 DELETE_HEALTH_CHECK_RESPONSE = """<?xml version="1.0" encoding="UTF-8"?>
     <DeleteHealthCheckResponse xmlns="https://route53.amazonaws.com/doc/2013-04-01/">
 </DeleteHealthCheckResponse>"""
+
+GET_CHANGE_RESPONSE = """<?xml version="1.0" encoding="UTF-8"?>
+<GetChangeResponse xmlns="https://route53.amazonaws.com/doc/2013-04-01/">
+   <ChangeInfo>
+      <Status>INSYNC</Status>
+      <SubmittedAt>2010-09-10T01:36:41.958Z</SubmittedAt>
+      <Id>/change/{{ change_id }}</Id>
```

Comment:
Looks like the test fails because of this, as the ID should just be {{change_id}} (without '/change/')

Happy to merge once Travis is happy.

Reply:
oh! that's definitely a typo! I'll fix that now.